ceph-releases/octopus/centos/7/__CEPH_MGR_PACKAGES__

```diff
@@ -0,0 +1 @@
+ceph-mgr__ENV_[CEPH_POINT_RELEASE]__
```

Comment:
i am pasting the commit message here, will amend it if it fails to answer your question.

> octopus is already a "python3 only" release, but centos7 is missing
> quite a lot of python3 depdendencies for almost all of ceph-mgr-*
> packages, like: ceph-mgr-rook and ceph-mgr-dashboard, the missing
> dependencies are:
> 
> - python3-jsonpatch
> - python3-routes
> - python3-remoto
> - python3-jinja2
> - python3-jwt
> - python3-kubernetes
> - python3-scipy
> - python3-cherrypy
> 
> since octopus built for RHEL/CentOS 7 is part of upgrade path, a fully
> functional dashboard is not our priority, but we do need ceph-mgr is
> critical. so let's just package ceph-mgr for octopus on el7.
>